tests/workflows/theiacov/test_wf_theiacov_illumina_pe.yml

```diff
@@ -0,0 +1,12 @@
+- name: theiacov_illumina_pe_cromwell
+  command: cromwell run -i ./tests/inputs/theiacov/wf_theiacov_illumina_pe.json -m metadata.json ./workflows/theiacov/wf_theiacov_illumina_pe.wdl
+  tags:
+    - wf_theiacov_illumina_pe
+    - wf_theiacov_illumina_pe_cromwell
+  files:
+    - path: log.err
+      contains: ["Successfully released change log lock", "INFO", "WORKFLOW_METADATA_SUMMARY_ENTRY", "liquibase"]
+    - path: log.out
+      contains: ["workflow finished with status 'Succeeded'", "theiacov_illumina_pe", "Done"]
+    - path: metadata.json
+      contains: ["outputs", "theiacov_illumina_pe", "Succeeded"]
```

Comment:
Any issue with including a command block to check proper execution and file outputs similar [PHVG illumina_pe test](https://github.com/theiagen/public_health_viral_genomics/blob/main/tests/workflows/test_illumina_pe.yml#LL14C4-L14C4)?

Reply:
Ah, I see the comment RE incompatibilities. Are you able to elaborate on this? Odd that there's no issues with the other tests.